src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -1908,7 +2041,7 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
                 std::vector<llvm::Value*> idx_vec = {idx};
                 p = CreateGEP(str, idx_vec);
             } else {
-                p = lfortran_str_item(str, idx);
+                p = lfortran_str_copy(str, idx, idx);
```

Comment:
to be reverted (https://github.com/lcompilers/lpython/commit/2f45421c06c2bee6a9918ae71ca0b88530ebd6fd)